package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "browserify": "^13.0.0",
     "d3": "3.5",
+    "lodash.result": "^4.4.0",
```

Comment:
I think we should stick to a no-dependency approach to keep CF as lightweight as possible, generally speaking. What would it look like to implement this without using lodash?